src/dotnet/APIView/APIViewWeb/Startup.cs

```diff
@@ -105,7 +105,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddSingleton<LanguageService, SwiftLanguageService>();
             services.AddSingleton<LanguageService, XmlLanguageService>();
 
-            if (Environment.IsDevelopment()) 
+            if (Environment.IsDevelopment() && Configuration["AuthenticationScheme"] == "Test")
```

Comment:
it this change related to diff? I don't see tests for this feature so not sure why we are adding this

Reply:
This is just a switch to make sure that the test authentication scheme is not used unless explicitly specified.